liferay-lima/liferay-portal
# LPS-182756 Create Dropdown menu for categories and the modal

https://issues.liferay.com/browse/LPS-182756

It's under FF LPS-84424

To test :
Create some Categories 
Go to Documents & Media
You can see a modal with the categories from the company

- LPS-182756 add categories dropdown element
- LPS-182756 create the category modal
- LPS-182756 buildLang

<img width="270" alt="image" src="https://github.com/liferay-lima/liferay-portal/assets/47524751/a5a4d088-c43a-4f3e-ae0c-52dd0035024a">
<img width="752" alt="image" src="https://github.com/liferay-lima/liferay-portal/assets/47524751/f50c99e9-8e03-4709-9e9c-582971aed323">
